### PR TITLE
Document update for new allow_email_update_only_from_manage config

### PR DESCRIPTION
**Pull request by jashaik**
Signed-off-by: jan shahid shaik <jashaik@progress.com>

### Description

Documentation update for new allow_email_update_only_from_manage config

### Issues Resolved
#2545 

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: docs-chef-io/content/server/v14/config_rb_server_optional_settings.md [hunk 1]
```diff
@@ -1289,6 +1289,15 @@ This configuration file has the following settings for `opscode-erchef`:
 
     **New in Chef Infra Server 14.1**
 
+`opscode_erchef["allow_email_update_only_from_manage"]`
+
+:   Allow preventing user's from changing their e-mail with new allow_email_update_only_from_manage config.
```
Comment (PrajaktaPurohit) [inline docs-chef-io/content/server/v14/config_rb_server_optional_settings.md:1294]: When set to true will allow users to update their email only from the manage console. When set to false users can change email from manage console as well as knife. (or something to this effect)

File: docs-chef-io/content/server/v14/config_rb_server_optional_settings.md [hunk 2]
```diff
@@ -1289,6 +1289,15 @@ This configuration file has the following settings for `opscode-erchef`:
 
     **New in Chef Infra Server 14.1**
 
+`opscode_erchef["allow_email_update_only_from_manage"]`
+
+:   When set to true will allow users to update their email only from the manage console.
+    When set to false users can change email from manage console as well as knife.
```
Comment (tas50) [inline docs-chef-io/content/server/v14/config_rb_server_optional_settings.md:1295]:
```suggestion
    When set to false users can change email from the Manage UI as well as knife.
```

File: docs-chef-io/content/server/v14/config_rb_server_optional_settings.md [hunk 3]
```diff
@@ -1289,6 +1289,15 @@ This configuration file has the following settings for `opscode-erchef`:
 
     **New in Chef Infra Server 14.1**
 
+`opscode_erchef["allow_email_update_only_from_manage"]`
+
+:   When set to true will allow users to update their email only from the manage console.
+    When set to false users can change email from the Manage UI as well as knife.
```
Comment (IanMadd) [inline docs-chef-io/content/server/v14/config_rb_server_optional_settings.md:1295]:
```suggestion
:   When set to `true`, users can only update their email from the Chef management console.
    When set to `false`, users can update their email using knife and the Chef management console.
```

**Comment by IanMadd** [comment]
Can we bump this definition to the top of the list (above `opscode_erchef['auth_skew']`) so it's in alphabetical order.

**Review by PrajaktaPurohit — APPROVED** [review]
Thank you!

**Comment by kagarmoe** [comment]
@IanMadd Can we rephrase this to use "Set to" (the indicative tense) instead of "When set to" (the conditional tense)? This makes it clearer and shorter.

**Comment by jashaik** [comment]
> @IanMadd Can we rephrase this to use "Set to" (the indicative tense) instead of "When set to" (the conditional tense)? This makes it clearer and shorter.

Done

**Review by IanMadd — APPROVED** [review]
